src/components/DayCard.js

```diff
@@ -41,14 +40,14 @@ const DayCard = ({ day, isActive }) => {
             <CardColumns>{date}</CardColumns>
             <CardColumns>{time}</CardColumns>
           </CardSubtitle>
-          <CardText className="d d-flex justify-content-between mt-2 m-1">
+          <CardColumns className="d d-flex justify-content-between mt-2 m-1">
```

Comment:
is className `d` doing anything or is it by mistake?

Reply:
d and d-flex is used together